data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
I'm wondering if we want to set a "single switch" for the entire application of which CEJST definition of DACs (e.g., `Definition M`) is currently live in the app. For instance, the front-end code could be updated to branch as well so that "if Def M is live, show these fields in the side bar, and if Def N is live, show this in the sidebar". That would make it easier to update the user interface and backend in a single deploy by switching the name of the definition of DACs that is active.

Reply:
Per convo with @lucasmbrown-usds, the versioning functionality for both frontend and backend will come in a separate release that will include both.